airflow/www/views.py

```diff
@@ -1049,21 +1102,22 @@ def trigger(self, session=None):
 
         dr = DagRun.find(dag_id=dag_id, execution_date=execution_date, run_type=DagRunType.MANUAL)
         if dr:
+            # noinspection PyUnresolvedReferences
             flash(f"This run_id {dr.run_id} already exists")
             return redirect(origin)
 
         run_conf = {}
-        conf = request.values.get('conf')
-        if conf:
+        the_conf = request.values.get('conf')
```

Comment:
```suggestion
        request_conf = request.values.get('conf')
```
Or something more informative also will be good

Reply:
Yep.